fiboa_cli/datasets/us_usda_cropland.py

```diff
@@ -0,0 +1,96 @@
+from os.path import join, dirname
+
+from .commons.ec import load_ec_mapping
+from ..util import log
+from ..convert_utils import convert as convert_
+import pandas as pd
+
+
+SOURCES = {
+    "https://www.nass.usda.gov/Research_and_Science/Crop-Sequence-Boundaries/datasets/NationalCSB_2016-2023_rev23.zip": ["NationalCSB_2016-2023_rev23/CSB1623.gdb"]
+}
+ID = "us_usda_cropland"
+SHORT_NAME = "USDA CSB"
+TITLE = "USDA Crop Sequence Boundaries"
```

Comment:
```suggestion
ID = "us_usda_cropland"
SHORT_NAME = "US (USDA CSB)"
TITLE = "U.S. Department of Agriculture Crop Sequence Boundaries"
```

I had no clue what USDA is, so maybe to make it more clear and to align with the California dataset like this?